spec/requests/line_items/create_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe "LineItems" do
+  
+  describe "create line item" do
+
+    let!(:product) { FactoryBot.create(:product, :with_image, name: 'Ipod', price: 120) }
+    let!(:user) { FactoryBot.create(:user) }
+    let!(:cart) { FactoryBot.create(:cart) }
+    let(:params) do
+        {
+          line_item: {
+            product: product,
+            cart: cart,
+            quantity: 2,
+            total: 240
+          }
+        }
+    end
+
+    before do
+      sign_in user
+      post line_items_path(product_id: product), params: params
+    end
+
+    it { expect(response).to have_http_status(302) }
```

Comment:
Add some a test that checks if the line item record actually has been created with the values passed through params, check tunelweb create tests as an example